facebook/OpenBIC
# op2: gl: Change SMI polling time to 500ms

# Description
- Chagne SMI event polling time from 1s to 500ms.

# Motivation
- Avoid send fake SMI event to BMC.

# Test plan
- Build code: Pass
- Check SMI event: Pass
